tests/lib/wifi.py

```diff
@@ -177,3 +178,165 @@ def uciSetWifiSecurity(board, vap_iface, security):
+
+def get_channel_util(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device, board,iface_ip,mib_name ,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_operating_mode(device, board, iface_ip, mib_name, index, set_type, operating_mode):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,operating_mode)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def get_operating_mode(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name ,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_bandwidth(device, board, iface_ip, mib_name, index, set_type, bandwidth):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,bandwidth)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def get_bandwidth(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_channel_no(device, board, iface_ip, mib_name, index, set_type, channel_no):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,channel_no)
+    if mib_out != None:
+       return mib_out
+    else:
+       return None
+
+def get_channel_no(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_broadcast(device, board, iface_ip, mib_name, index, set_type, broadcast):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,broadcast)
+    if mib_out != None:
+       return mib_out
+    else:
+       return None
+
+def get_broadcast(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+class wifi_stub():
+    def enable_wifi_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_ssid_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_broadcast_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_security_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_password_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def enable_channel_util_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_operating_mode_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_bandwidth_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def set_channel_no_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
```

Comment:
This class looks ok- but why is everything appended with _verify?

Reply:
Since the function in device class does a set and verify of the values we appended with _verify. The same name has been used here. Should we remove this?